examples/advanced/bionemo/task_fitting/jobs/fedavg/app/custom/bionemo_mlp_learner.py

```diff
@@ -0,0 +1,329 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+import math
+import os
+import pickle
+from distutils.util import strtobool
+from typing import Union
+
+import numpy as np
+import pandas as pd
+import sklearn
+from sklearn.metrics import accuracy_score
+from sklearn.neural_network import MLPClassifier
+from torch.utils.tensorboard import SummaryWriter
+
+from nvflare.apis.fl_constant import FLMetaKey, ReturnCode
+from nvflare.app_common.abstract.fl_model import FLModel, ParamsType
+from nvflare.app_common.abstract.model_learner import ModelLearner
+from nvflare.app_common.app_constant import AppConstants, ModelName, ValidateType
+from nvflare.app_common.utils.fl_model_utils import FLModelUtils
+
+
+class BioNeMoMLPLearner(ModelLearner):  # does not support CIFAR10ScaffoldLearner
```

Comment:
just curious, can we use Client API here? why do we choose ModelLearner here?

Reply:
The example was developed before client API was available.